tests/test_jurisdiction.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+import pytest
+from freezegun import freeze_time
+
+
+@freeze_time('2024-06-23')
+def test_legislative_session(mocker):
+    '''
+    Test that next fiscal year's legislative sessions are included
+    when it's the last week of the current one.
+    '''
+    fake_now = datetime.now()
+    mocker.patch(
+        "lametro.Lametro.today", return_value=fake_now
```

Comment:
What if you mocked `datetime.now`, which is how the value of `today` is determined?

Reply:
Actually, @xmedr, no need to mock anything! Freezegun should inject the configured date universally. Does it work as expected if you remove the mock call?

Reply:
P.s., if you wanted to [parametrize your test](https://github.com/datamade/testing-guidelines/blob/master/intermediate-python-testing.md#parameterizing-fixtures) to cover the different cases, you can use `freeze_time` [as a context manager](https://github.com/spulec/freezegun?tab=readme-ov-file#context-manager)

Reply:
So whether I kept the mock or got rid of it, the datetime within the test gets changed which is great, but the datetime within the Lametro class is still the actual time. This is how it looks when checking things out with a breakpoint:

```
-> breakpoint()
(Pdb) datetime.now()
FakeDatetime(2024, 6, 23, 0, 0)
(Pdb) fake_now
FakeDatetime(2024, 6, 23, 0, 0)
(Pdb) Lametro.today
datetime.datetime(2024, 9, 11, 18, 48, 31, 188266)
```

And nice, that context manager bit is cool! I was actually hoping to parameterize the test once things start working

Reply:
Could this be because  `legislative_sessions` is a static class variable that we're accessing without instantiating an `Lametro` object?

Reply:
I think this was it! It must've been difficult to recalculate the sessions as a static variable because it was already set by the time we mocked things.

So I've changed the `legislative_sessions` static var into the `get_legislative_sessions()` static method instead. This way it can still get called without instantiating an `Lametro` object, while also encapsulating the calculation in a method as opposed to a free standing loop. So with a small adjustment to bills.py it still works fine, and can be affected by the frozen time in tests! Thank yall ☺️